string role_req
---
string robot_id
string robot_role
